Library/Formula/pv.rb

```diff
@@ -10,16 +10,24 @@ class Pv < Formula
     sha256 "f343368e557cb1c86173bd0c62143b34834e2b825b1a188ac2a37c23d0c685dd" => :mountain_lion
   end
 
-  depends_on "gettext"
+  option "with-nls", "Build with Native Language Support"
+
+  depends_on "gettext" if build.with? "nls"
```

Comment:
We've been trying to kill off `if build.with?` deps. Adding new ones is a bit bleh.

Agree with Mike we should make it `=> :optional` instead; If it's helpful we can make the option `option "with-gettext", "Build with Native Language Support"` so it's more explicit what it brings to the party.